index_impl.go

```diff
@@ -578,6 +604,14 @@ func (i *indexImpl) SearchInContext(ctx context.Context, req *SearchRequest) (sr
 					}
 					skipSynonymCollector = true
 				}
+				skipKNNCollector = true
```

Comment:
Is this right? Something we missed?

Reply:
yeah i've removed this, it seems like a merge conflict issue, thanks!